src/components/adminPollings/PollAnswers.tsx

```diff
@@ -0,0 +1,398 @@
+            </Button>
+          </FormGroup>
+        </form>
+        <Divider className={classes.divider} />
+      </Grid>
+      <Grid item xs={12}>
+        <Typography variant="h3">Answers to these question</Typography>
+        {data?.poll_answers.length === 0 ? (
+          <Alert severity="info">Please add an answer to the poll.</Alert>
+        ) : (
+          data?.poll_answers
+            .sort((a, b) => a.id - b.id)
+            .map((answer) => (
+              <FormGroup row key={answer.id}>
+                <Grid item xs={8}>
+                  <TextField
+                    key={answer.id}
+                    name={answer.text + answer.id}
+                    required
+                    disabled={getPollQuestion.data?.poll_question[0]?.is_active}
+                    onChange={(e) => {
+                      handleAnswerChange(answer?.id, e);
+                      setAnswerTextUpdateId(answer.id);
+                    }}
+                    onClick={() => {
+                      console.log('clicked');
+                      answerText.text = '';
+                    }}
+                    size="medium"
+                    variant="outlined"
+                    color="secondary"
+                    autoComplete="off"
+                    placeholder="Type your answers here ..."
+                    label={answer.text}
+                    fullWidth
+                    margin="dense"
+                    InputProps={{
+                      classes: {
+                        input: classes.messageInput,
+                      },
+                    }}
+                    InputLabelProps={{
+                      className: classes.messageInput,
+                    }}
+                  />
+                </Grid>
+                <Grid item xs={4}>
+                  <Box
+                    display="flex"
+                    justifyContent="flex-end"
+                    alignItems="center"
+                  >
+                    <Button
+                      style={{
+                        marginTop: '8px',
+                        marginLeft: '8px',
+                        maxWidth: '100px',
+                      }}
+                      key={answer.id}
+                      variant="contained"
+                      size="large"
+                      color="secondary"
+                      disabled={
+                        answer.id !== answerTextUpdateId
+                          ? true
+                          : false || updateEnabled === true
+                      }
+                      onBlur={() => {
+                        setUpdateEnabled(true);
+                        console.log(updateEnabled);
+                      }}
+                      onClick={() => {
+                        handleUpdateAnswerText(answer.id);
+                      }}
+                    >
+                      Update
+                    </Button>
+                    <Button
```

Comment:
Solche Dinge würde ich in eine separate Komponente auslagern. Z.b. eine für Edit, eine für Delete und so weiter, sonst wird die Komponente sehr gross. Du könntest dann den ganzen Button in die neue Komponente reinpacken...